Source/JavaScriptCore/heap/BlockDirectory.cpp

```diff
@@ -59,8 +59,9 @@ void BlockDirectory::setSubspace(Subspace* subspace)
 
 void BlockDirectory::updatePercentageOfPagedOutPages(SimpleStats& stats)
 {
-    // FIXME: We should figure out a solution for Windows.
-#if OS(UNIX)
+#if OS(WINDOWS) || PLATFORM(PLAYSTATION)
+    // FIXME: We should figure out a solution for Windows and PlayStation.
+#elif OS(UNIX)
```

Comment:
Even though we may want a common solution for Win & PS, I think it's probably best to just use `#if OS(UNIX) && !PLATFORM(PLAYSTATION)`?

Reply:
Probably Win's solution will be different from PS's so I'll change that condition.